pkg/functions/deploy.go

```diff
@@ -73,6 +73,45 @@ func RunDeploy(w io.Writer, sdk *SdkStatus) error {
 			if err != nil {
 				return err
 			}
+		} else if step.Build.Builder != "" {
+			if step.Build.Destination == "" {
+				return fmt.Errorf("Please provide a destination for the build image.")
+			}
+			if step.Build.MainPath == "" {
+				return fmt.Errorf("Please provide the relative path to the main file.")
+			}
+			packCommand := fmt.Sprintf("pack build %s --builder %s --env GOOGLE_BUILDABLE=%s --publish", step.Build.Destination, step.Build.Builder, step.Build.MainPath)
+			cmd := exec.Command("/bin/sh", "-c", packCommand)
+			cmd.Env = os.Environ()
+			cmd.Stdin = os.Stdin
+			cmd.Stdout = os.Stdout
+			cmd.Stderr = os.Stderr
+			if err := cmd.Run(); err != nil {
+				return err
+			}
+
+			data := make(map[string]interface{})
+			data["MetadataName"] = deployPlan.ObjectMeta.Name
+			data["Image"] = step.Build.Destination
+			sourceFile := fmt.Sprintf("/pkg/functions/template/service.yaml.tmpl")
```

Comment:
this location for that file won't work

If we want to include that w/ the binary, it needs to be part of the executable.
Something like https://github.com/gobuffalo/packr

The other templates in use so far are from the funk-sdk-go which is meant to be unpacked files on disk

Reply:
Hmm you're right. It worked great when i had the full path hardcoded, but now not so much.